docs/guides/upstream-dns-providers.md

```diff
@@ -17,10 +17,15 @@ During the pi-hole installation, you select 1 of the 7 preset providers or enter
 
 Default upstream DNS provider on the Pi-hole.
 
-- 8.8.8.8
-- 8.8.4.4
+- 8.8.8.8 (IPv4)
+- 8.8.4.4 (IPv4)
+- 2001:4860:4860::8888 (IPv6)
+- 2001:4860:4860::8844 (IPv6)
+- 2001:4860:4860::6464 (IPv6 with DNS64)
+- 2001:4860:4860::64 (IPv6 with DNS64)
```

Comment:
The PR that added the DNS64 servers was declined. Please remove the references to these servers.